src/components/Doc/DomDocs.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Box, Paragraph, Anchor, Heading } from 'grommet';
+import { Link as LinkIcon } from 'grommet-icons';
+
+export const DomDocs = ({ name, intrinsicElement }) => {
+  return (
+    <Box
+      id="dom-react"
+      margin={{ vertical: 'large' }}
+      border={{ side: 'top', size: 'medium', color: 'brand' }}
+    >
+      <Box
+        direction="row"
+        justify="between"
+        align="center"
+        margin={{ top: 'medium', bottom: 'xlarge' }}
+      >
+        <Heading level={2} margin="none">
+          React/DOM Properties
+        </Heading>
+        <Anchor href="#dom-react" icon={<LinkIcon color="light-4" />} />
+      </Box>
+      <Paragraph>
+        At its core, the {name} component is a regular {`<${intrinsicElement}>`}{' '}
+        element. Thus, both DOM and React properties, methods,
+        and events are accessible. To read up on all of the possible DOM
+        attributes and types available for {intrinsicElement} elements, check
+        out this{' '}
+        <Anchor
+          href={`https://developer.mozilla.org/en-US/docs/Web/HTML/Element/${intrinsicElement}`}
+          label="MDN Web Documents page."
+          target="_blank"
+        />{' '}
+        To learn more about DOM events and methods, you can read more{' '}
+        <Anchor
+          href="https://developer.mozilla.org/en-US/docs/Web/Events"
+          label="here."
```

Comment:
```suggestion
          label="MDN Web Documents Events page"
```
I think you might've had this on a previous iteration, but it's better for screen readers if the Anchor text is descriptive about where it's going. I'd recommend rephrasing to something more focused.